<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MLBridge</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>960</width>
     <height>486</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="mainLayout" stretch="0,25">
    <item>
     <layout class="QVBoxLayout" name="foldersLayout">
      <item>
       <widget class="QListWidget" name="listWidgetModel"/>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetData"/>
      </item>
      <item>
       <widget class="QPushButton" name="actionRefresh">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
        </property>
        <property name="shortcut">
         <string>Ctrl+S, Ctrl+S</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabMain">
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabInference">
       <attribute name="title">
        <string>Предсказывание</string>
       </attribute>
       <widget class="QPushButton" name="startInference">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>151</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Анализировать данные</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveResults">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>151</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Сохранить результат</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="inferenceLogs">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>651</width>
          <height>271</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>251</width>
          <height>50</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="infoInferenceLayout" stretch="0,0">
         <item>
          <widget class="QLabel" name="chosedModelInference">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Panel</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="text">
            <string>Выбранная модель</string>
           </property>
           <property name="textFormat">
            <enum>Qt::TextFormat::AutoText</enum>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="chosedDataInference">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Panel</enum>
           </property>
           <property name="text">
            <string>Выбранные данные</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>651</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="infWithLabels">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>60</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>С метками</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionLoadModel"/>
    <addaction name="actionLoadData"/>
   </widget>
   <widget class="QMenu" name="menuDocs">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Папки</string>
    </property>
    <addaction name="actionOpenResults"/>
    <addaction name="actionOpenAppFolder"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menuDocs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadData">
   <property name="text">
    <string>Загрузить данные...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionToggleLogs">
   <property name="text">
    <string>Показать/скрыть логи</string>
   </property>
  </action>
  <action name="actionDefaultPath">
   <property name="text">
    <string>Папка по умолчанию...</string>
   </property>
  </action>
  <action name="actionModelOpts">
   <property name="text">
    <string>Параметры модели...</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Документация</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionLoadZip">
   <property name="text">
    <string>.zip</string>
   </property>
  </action>
  <action name="actionLoadFolder">
   <property name="text">
    <string>Папка</string>
   </property>
  </action>
  <action name="actionLoadModel">
   <property name="text">
    <string>Загрузить модель...</string>
   </property>
  </action>
  <action name="actionOpenResults">
   <property name="text">
    <string>Открыть папку с результатами...</string>
   </property>
  </action>
  <action name="actionOpenAppFolder">
   <property name="text">
    <string>Открыть папку с программой...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
